### PR TITLE
Render service sample render

### PR DESCRIPTION
**Pull request by NISHIY-EKSDEE**
### Jira Ticket
https://adc.luxoft.com/jira/browse/STVCIS-1300

### Purpose
Make sample render with low options to check if scene is correct and can be rendered.

### :warning: Notes for Reviewers
- Changed python3 implementation due to Baku error, now it works even there
- I'm not sure how to handle incorrect scenes. Now, if scene is incorrect, build is failed. The essence of the pipeline is in catching broken scenes, so sample render failure may be a success result. On other side, there may occur a machine error, so build must be failed. Is there any reason to implement this or failed builds are ok for us?
- Now if scene is incorrect, we continue trying to render it on other machine (I left it due to machine errors on Baku or Kabul). Should I stop the job after failed sampled render?
### :octocat: Related PR'S
https://gitlab.cis.luxoft.com/services/render_service/-/merge_requests/60
https://github.com/luxteam/render_service_scripts/pull/21

### CODE REVIEW

File: vars/render_service_scene_configuration.groovy [hunk 1]
```diff
@@ -1,231 +1,268 @@
...
+                } catch(FlowInterruptedException e) {
+                    throw e
+                } catch(e) {
+                    fail_reason = "Downloading scripts failed"
+                    throw e
+                }
+
+                // download and install plugin
+                if (options["pluginLink"]) {
+                    render_service_install_plugin(options["pluginLink"], options["pluginHash"], tool, "2.83", options.id, options.django_url)
```
Comment (Speedwag00n) [inline vars/render_service_scene_configuration.groovy:33]: Плагин инсталлится на 2.83 Blender, а не на ту версию блендера, которую указали

File: vars/render_service_scene_configuration.groovy [hunk 2]
```diff
@@ -1,231 +1,268 @@
...
+                        case 'Blender':
+                            if (options["id_render"] && options["django_render_url"]) {
+                                try {
+                                    bat """
+                                        copy "render_service_scripts\\blender_render.py" "."
+                                        copy "render_service_scripts\\launch_blender.py" "."
+                                    """
+                                    // Launch render
+                                    withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'renderServiceCredentials', usernameVariable: 'DJANGO_USER', passwordVariable: 'DJANGO_PASSWORD']]) {
+                                        python3("launch_blender.py --tool \"2.91\" --django_ip \"${options.django_render_url}/\" --scene_name \"${scene_name}\" --id ${options.id_render} --min_samples 16 --max_samples 32 --noise_threshold 0.1 --height 150 --width 150 --startFrame 1 --endFrame 1 --login %DJANGO_USER% --password %DJANGO_PASSWORD% --timeout 120 ")
```
Comment (Speedwag00n) [inline vars/render_service_scene_configuration.groovy:81]: Тут тоже захардкожена версия Блендера

**Review by Speedwag00n — CHANGES_REQUESTED** [review]

File: vars/render_service_scene_configuration.groovy (hunk 2 above)
Comment (NISHIY-EKSDEE) [reply vars/render_service_scene_configuration.groovy]: Добавил версию Блендера, который используется для конфигурации, в blender_config \
https://gitlab.cis.luxoft.com/services/render_service/-/merge_requests/60/diffs#1a19dbd43fbd26c4cd98c427809b809e06d51435_6_6
Передаю ее вместе с параметром tool, как это сейчас сделано в рендер джобах

**Review by Speedwag00n — APPROVED** [review]

File: vars/render_service_scene_configuration.groovy [hunk 3]
```diff
@@ -1,231 +1,269 @@
...
-
-	if (!successfullyDone) {
-		node("RenderService") {
-			render_service_send_render_status('Failure', options.id, options.django_url)
-		}
-		throw new Exception("Job was failed by all used nodes!")
-	} else {
-		currentBuild.result = 'SUCCESS'
-	}
+    node("RenderService || Windows && Builder") {
```
Comment (epozine) [inline vars/render_service_scene_configuration.groovy:156]: Может лучше httpRequest использовать, чем занимать ноду?

Comment (epozine) [reply vars/render_service_scene_configuration.groovy:156]: https://github.com/luxteam/rpr_pipelines/blob/master/src/universe/UniverseClient.groovy#L283
Пример

Comment (NISHIY-EKSDEE) [reply vars/render_service_scene_configuration.groovy]: Убрал блок node перед отправкой номера билда и статуса, все работает
https://rpr.cis.luxoft.com/job/RenderService/job/TestRenderServiceSceneConfigurationJob/127/
https://rpr.cis.luxoft.com/job/RenderService/job/TestRenderServiceSceneConfigurationJob/128/

**Review by epozine — APPROVED** [review]

**Pull request merged**